File: 프로그래머스/프로그래머스_고득점/정렬/K번째수.py
def solution(array, commands):
    answer = []

    for i in range(len(commands)):
        pre = []
        for j in range(commands[i][0] - 1, commands[i][1]):
            pre.append(array[j])
        pre.sort()
        answer.append(pre[commands[i][2]-1])
    return answer

File: java/프로그래머스/고득점kit/정렬/k번째수.java
package programmers;

import java.util.*;

public class k번째수 {
    class Solution {
        public int[] solution(int[] array, int[][] commands) {
            int[] answer = new int[commands.length];

            for (int i = 0; i < commands.length; i++) {

                int[] impl = new int[commands[i].length];
                int loc = 0;
                for (int j = commands[i][0]; j <= commands[i][1]; j++) {
                    impl[loc] = array[j - 1];
                    loc += 1;
                }

                Arrays.sort(impl);
                answer[i] = impl[commands[i][2]];

            }

            return answer;
        }
    }
}
